tests/filtertransformers/test_mongo.py

```diff
@@ -341,8 +342,170 @@ def test_not_implemented(self):
                 'HAS ANY > 3:"He":>55.3 , = 6:>"Ti":<37.6 , 8:<"Ga":0'
             )
 
-        with self.assertRaises(VisitError):
-            self.transform("list LENGTH > 3")
+        self.assertEqual(
+            self.transform("list LENGTH > 3"), {"list.4": {"$exists": True}}
+        )
+
+    def test_list_length_aliases(self):
+        from optimade.server.mappers import StructureMapper
+
+        transformer = MongoTransformer(mapper=StructureMapper())
+        parser = LarkParser(version=self.version, variant=self.variant)
```

Comment:
I guess you don't need to define this again, as it's defined already as `p` in line 15?

Reply:
See answer below